enclone_print/src/lib.rs

```diff
@@ -1,7 +1,3 @@
-// Copyright (c) 2020 10x Genomics, Inc. All rights reserved.
-
-extern crate enclone_core;
-
```

Comment:
```suggestion
// Copyright (c) 2020 10x Genomics, Inc. All rights reserved.
```
I believe `enclone_print` should still depend on `enclone_core`... but if all is okay during `cargo t` no objections

Reply:
Good catch. Added the copyright back.

Re `extern crate enclone_core;`: Since rust 2018 you don't need the extern crate explicitly.